roles/edpm_build_images/tasks/post.yaml

```diff
@@ -12,7 +12,8 @@
     - ironic-python-agent
 
 - name: Retag and push the images with podified-ci-testing tag
-  when: cifmw_repo_setup_promotion == "podified-ci-testing"
+  when: cifmw_repo_setup_promotion
+        in ("podified-ci-testing", "podified-ci-testing-tcib")
```

Comment:
I don't know if tuple is ok, or it should not be a list...

Reply:
ok, it seems jinja2 accept tuple

Reply:
It works in testproject, so Ansible seems fine with Tuples :wink: